apps/mark-scan/frontend/src/app.tsx

```diff
@@ -149,14 +159,19 @@ export function App({
           >
             <ApiClientContext.Provider value={apiClient}>
               <QueryClientProvider client={queryClient}>
-                <AppRoot
-                  hardware={hardware}
-                  storage={storage}
-                  screenReader={screenReader}
-                  reload={reload}
-                  logger={logger}
-                />
-                <SessionTimeLimitTracker />
+                <UiStringsContextProvider
+                  api={uiStringsApi}
+                  disabled={!enableStringTranslation}
```

Comment:
The positive-to-negative mapping here feels admittedly awkward - initially had a `disableStringTranslationForTesting` prop on the `<App>` component, but that would've meant requiring every test using `<App>` to remember to explicitly disable, vs explicitly enabling once in `index.tsx`.

Reply:
Ah yes the art of enable/disable params haha 🎨

After taking a sec to reason through the various possible permutations of params on `App` and `UiStringsContextProvider`, I like what you've landed on